src/jackdaw/streams/specs.clj

```diff
@@ -49,7 +49,8 @@
 (s/def ::topic-config
   (s/keys :req-un [::topic-name
                    ::key-serde
-                   ::value-serde]))
+                   ::value-serde]
+          :opt-un [::partition-fn]))
```

Comment:
Can we add this to the specs in `jackdaw.specs` too?

Reply:
will do